power_grid_model_io_native_c/power_grid_model_io_native_c/src/c_converter_api.cpp

```diff
@@ -0,0 +1,21 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#define PGM_IO_DLL_EXPORTS
+
+#include "../../../../power_grid_model_io_native_c/power_grid_model_io_native_c/include/power_grid_model_io_native_c/c_converter_api.h"
+#include "../../../../power_grid_model_io_native_c/power_grid_model_io_native_c/include/power_grid_model_io_native_c/vnf_pgm_converter.h"
+#include "handle.hpp"
+
+#include "../../../../../power-grid-model/power_grid_model_c/power_grid_model_c/src/forward_declarations.hpp"
```

Comment:
this is way too many `../`. you shouldn't need any of them in this particular case if your `target_include_directories` and/or `target_link_libraries` are correct